src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -139,8 +139,14 @@ WaveTrackAffordanceControls::WaveTrackAffordanceControls(const std::shared_ptr<T
 {
     if (auto trackList = pTrack->GetOwner())
     {
-        mSubscription = trackList->Subscribe(
-            *this, &WaveTrackAffordanceControls::OnTrackChanged);
+        if(auto project = trackList->GetOwner())
+        {
+            auto& viewInfo = ViewInfo::Get(*project);
+            mSelectionChangeSubscription =
+                viewInfo.selectedRegion.Subscribe(
+                    *this,
+                    &WaveTrackAffordanceControls::OnSelectionChange);
+        }
```

Comment:
This adds detection of changes in the selected interval of time, but removes detection of changes of the binary selection state of tracks.

Why don't we do both, with two subscriptions?